src/IB/IBFEMethod.cpp

```diff
@@ -3492,7 +4157,17 @@ IBFEMethod::commonConstructor(const std::string& object_name,
 
     // Indicate that all of the parts do NOT use stress normalization by default
     // and set some default values.
-    d_epsilon = 0.0;
+    d_phi_epsilon = 0.0;
```

Comment:
You _might_ want to group these together into `struct`s, to make the name spacing "nicer".